### ISSUE TITLE
Change Sort By

### ISSUE THREAD

**Issue by shanehayes** [jeffpriz/ado-wiki-age-report#4]
Hi,

We downloaded your extension, which is great but we are wanting to use it the opposite way, where by we want to display recently created articles first so that people in the project know about new wiki articles. 
Can you add configuration/option to change the sort order  Oldest > Newest updated  to Newest Updated > Oldest 
![image](https://user-images.githubusercontent.com/77268025/151556404-ca0f4fd9-ff5c-4ef9-9d95-bf2f164fc38d.png)

**Comment by jeffpriz** [comment]
Yeah that's a great suggestion.  I can work to make the table columns sortable.  that way you could change the sort dynamically.
